README.rst

```diff
@@ -16,7 +16,9 @@ and can be installed via ``pip`` or ``easy_install``:
 
 Requirements
 ------------
-- sprockets.mixins.amqp>=1.0.0
+- sprockets.mixins.amqp>=2.0.0
+- fastavro==0.10.1
```

Comment:
Is there a reason you hard-pinned fastavro? Since this is a library, we typically loose pin dependencies.